suite/tests/api/ir_x86_all_opc.h

```diff
@@ -57,7 +57,7 @@
 #   undef OPCODE
 #   undef XOPCODE
 
-    end = instrlist_encode(dc, ilist, buf, false);
+    end = instrlist_encode(dc, ilist, buf, true);
```

Comment:
Hmm, interesting: yes, the cti instrs are given instr opnds, for simplicity I guess rather than making up some PC.  They are never executed so everything passed before.